packages/@okta/vuepress-site/docs/guides/index.md

```diff
@@ -30,7 +30,10 @@ guides:
  - implement-grant-type
  - mfa
  - migrate-to-okta
- - oin-oidc-guide
+ - oin-oidc-overview
```

Comment:
@chrisdmills-okta - this is the file where you need to add new guide names...now the links work in the files without having to add `/main/`